CHANGELOG.rst

```diff
@@ -23,11 +23,21 @@ Fixed
 Security
 ========
 
+[1.1.1] - 2019-03-12
+********************
+Fixed
+=====
+- Fixing linter issues and improve code organization.
+Added
+=====
+- Improve status napp.
+- Allow to run linter and tests on scrutinizer.
+
```

Comment:
Please move changelog to the UNRELEASED section until we make a real version release.

Reply:
Thanks. Solved